objects/src/testing/account_code.rs

```diff
@@ -49,80 +49,110 @@ impl AccountCode {
         export.::miden::contracts::wallets::basic::create_note
         export.::miden::contracts::wallets::basic::move_asset_to_note
 
-        export.::miden::account::get_item_foreign
-        export.::miden::account::get_map_item_foreign
+        ### Note: all account's export procedures below should be only called or dyncalled, so it is
+        ### assumed that the operand stack at the beginning of their execution is paded and doesn't 
+        ### have any other valuable information.
 
+        # Stack:  [value, pad(15)]
+        # Output: [pad(16)]
         export.incr_nonce
-            push.0 swap
-            # => [value, 0]
             exec.account::incr_nonce
-            # => [0]
+            # => [pad(16)]
         end
 
+        # Stack:  [index, VALUE_TO_SET, pad(11)]
+        # Output: [NEW_STORAGE_ROOT, PREVIOUS_STORAGE_VALUE, pad(8)]
         export.set_item
             exec.account::set_item
-            # => [R', V, 0, 0, 0]
+            # => [R', V, pad(11)]
+
+            # truncate the stack
             movup.8 drop movup.8 drop movup.8 drop
-            # => [R', V]
+            # => [R', V, pad(8)]
         end
 
+        # Stack:  [index, pad(15)]
+        # Output: [VALUE, pad(12)]
         export.get_item
             exec.account::get_item
+            # => [VALUE, pad(15)]
+
+            # truncate the stack
             movup.8 drop movup.8 drop movup.8 drop
+            # => [VALUE, pad(12)]
         end
```

Comment:
This is the procedure whose index is assumed to be 6, right? I would add a comment here that sequencing is important here.

Reply:
I'm not sure that we need to add indexes here in the mock code. It will be really annoying to keep them updated, since we will need to do that manually every time the code has changed. We use this index only in our FPI tests, so I think it should be sufficient to add comments and tests as you suggested above.